packages/Button/src/index.tsx

```diff
@@ -1,32 +1,34 @@
-/* eslint-disable react/button-has-type */
-import React from "react";
+import withUtils from "@blaze-react/utils";
+import React, { ButtonHTMLAttributes, FunctionComponent } from "react";
 
-interface IButtonProps {
+interface IButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
   disabled?: boolean;
-  submit?: boolean;
   children?: JSX.Element | string;
-  modifiers?: string;
-  attrs?: any;
+  type?: "button" | "submit" | "reset";
+  modifiers: string;
```

Comment:
This should be type as the parameters in the docs and also could be in static method, don't you think so?

![image](https://user-images.githubusercontent.com/24245588/61723090-c6cde900-ad6b-11e9-93e1-0c27c9c8155f.png)